app/views/notification_mailer/cinstance_plan_changed.html.slim

```diff
@@ -14,4 +14,4 @@ ul
   li Previous plan: #{@old_plan.name}
   li Current plan: #{@new_plan.name}
 
-p= link_to 'View application', admin_buyers_account_application_url(@account, @cinstance)
+p= link_to 'View application', admin_service_application_url(@cinstance.service, @cinstance, host: @account.provider_account.admin_domain)
```

Comment:
I do not think we need the host

https://github.com/3scale/porta/blob/master/app/mailers/notification_mailer.rb#L408-L410